### PR TITLE
media: Add SpeechDetector::startEndPointDetect

### PR DESCRIPTION
**Pull request by junmin-kim**
Since SpeechDetector::initEndPointDetect waits EPD, split it into init
and start

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ebae345a1624a4b2cdaaca0b52e4eb8827c393ba] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ebae345a1624a4b2cdaaca0b52e4eb8827c393ba] - Code Rule Check Result:
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/voice/HardwareEndPointDetector.h:28: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/voice/SoftwareEndPointDetector.h:41: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)

File: framework/src/media/voice/SoftwareEndPointDetector.cpp [hunk 1]
```diff
@@ -45,6 +38,14 @@ void SoftwareEndPointDetector::deinit()
 	speex_preprocess_state_destroy(mState);
 }
 
+bool SoftwareEndPointDetector::startEndPointDetect(uint32_t timeout)
+{
+	int adjust = 99;
```
Comment (thapav) [inline framework/src/media/voice/SoftwareEndPointDetector.cpp]: You could use #define for these values.

File: framework/src/media/voice/SoftwareEndPointDetector.cpp [hunk 2]
```diff
@@ -45,6 +38,14 @@ void SoftwareEndPointDetector::deinit()
...
+	speex_preprocess_ctl(mState, SPEEX_PREPROCESS_SET_PROB_START, &adjust);
+	adjust = 80;
```
Comment (thapav) [inline framework/src/media/voice/SoftwareEndPointDetector.cpp]: same as above

**Comment by seinfra** [comment]
Target : [7164d0715485cc7ac33b5d7ff969174e54845fe1] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7164d0715485cc7ac33b5d7ff969174e54845fe1] - Code Rule Check Result:
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/voice/SpeechDetector.h:38: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/voice/HardwareEndPointDetector.h:28: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/voice/SoftwareEndPointDetector.h:41: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)

**Pull request merged**